src/components/PlanRoom/SaveBoard.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, ButtonGroup, Box } from '@chakra-ui/react';
+import breakpointsButton from 'styled/PlanRoom/breakpointsButton';
+import breakpointsFontSize from 'styled/PlanRoom/breakpointsFontSize';
+import { NewTable, SaveTables } from './RoomTables/Item';
+
+const saveBoard = () => (
+  <Box h="4em" border="1px solid black">
+    <ButtonGroup variant="outline" spacing="6" px="2em" py="0.8em">
+      <Box minWidth="10%">
+        <Button
+          type="submit"
+          bg="#33D268"
```

Comment:
We should customize colors used in the app by using [Chakra Theme](https://chakra-ui.com/docs/theming/customize-theme#customizing-theme-tokens).
Please refer to the documentation and extend theme with concrete color / colors with proper naming convention.

Please apply it in all appropriate places so I don't have to write the same comment everywhere.

Reply:
will be changed